his/core/as-400-appc-printing1.md

```diff
@@ -8,16 +8,16 @@ ms.reviewer: ""
 ms.suite: ""
 ms.topic: "article"
 ---
-# AS/400 (APPC) Printing
-APPC printing, like LU 1 printing, uses SCS control codes in the data stream. The set of SCS control codes available for use in APPC printing is more extensive, and allows more formatting options than the set described for LU 1 printing. The IBM AS/400 also provides an additional method for formatting print jobs called Host Print Transform (HPT). With HPT enabled, the AS/400 takes responsibility for rendering the print job into data the printer can understand.  
+# IBM System i (APPC) Printing
+APPC printing, like LU 1 printing, uses SCS control codes in the data stream. The set of SCS control codes available for use in APPC printing is more extensive, and allows more formatting options than the set described for LU 1 printing. The IBM IBM System i also provides an additional method for formatting print jobs called Host Print Transform (HPT). With HPT enabled, the IBM System i takes responsibility for rendering the print job into data the printer can understand.  
   
- ![Image that shows the APPC connection between AS/400 and Host Integration Server to send print job to local printer.](../core/media/prn04.gif "prn04")  
-APPC connection between AS/400 and Host Integration Server to send print job to local printer  
+ ![Image that shows the APPC connection between IBM System i and Host Integration Server to send print job to local printer.](../core/media/prn04.gif "prn04")  
+APPC connection between IBM System i and Host Integration Server to send print job to local printer  
```

Comment:
This incomplete sentence looks like a caption for the image? If yes, doesn't belong after the image, but should appear preceding the image and reworded like this:

`The following diagram shows the APPC connection between IBM System i and Host Integration Server for sending a print job to the local printer:`